webapp/src/index.tsx

```diff
@@ -10,13 +10,14 @@ import {PluginRegistry} from './types/mattermost-webapp';
 export default class Plugin {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
     public async initialize(registry: PluginRegistry, store: Store<GlobalState, Action<Record<string, unknown>>>) {
-        // @see https://developers.mattermost.com/extend/plugins/webapp/reference/
+        window.store = store;
```

Comment:
Propose something a little more unique, just in case this collides with plugins in some unexpected way.

Reply:
Yep, good call